bridges/WikipediaBridge.php

```diff
@@ -107,9 +107,18 @@ private function AddElementGeneric($element, $fullArticle){
 		// Clean the bottom of the featured article
 		$element->find('div', -1)->outertext = '';
 
+		// The title and URI of the article is best defined in an anchor containint the string '...' ('full article ...')
+		$target = $element->find('p/a', 0); // We'll use the first anchor as fallback
+		foreach($element->find('//a') as $anchor){
+			if(strpos($anchor->innertext, '...') !== false){
```

Comment:
Your solution would solve that. Here is a slightly different approach, where the function call needs to be changed slightly:

```PHP
private function AddElementGeneric($element, $fullArticle, $anchorText = '...')
```

That way every language can specify their own text to search for and for those which work with the default anchor, nothing needs to be changed.

In case the fallback needs to be different, maybe even add another parameter:

```PHP
private function AddElementGeneric($element, $fullArticle, $anchorText = '...', $anchorFallbackIndex = 0)
```

`$anchorFallbackIndex` is an integer which allows us to specify the fallback position of the anchor. Again for those working with the default anchor, nothing needs to be changed:

```PHP
  $target = $element->find('p/a', $anchorFallbackIndex);
  foreach($element->find('//a') as $anchor){
    if(strpos($anchor->innertext, $anchorText) !== false){
```